### DISCUSSION TITLE
Gnudb not anymore support freac 1.1.7 since 2023/10/23

### DISCUSSION THREAD

**Discussion by madum31** [enzo1982/freac#550] [Help & Support]
Hello,

   I was surprised that no more pertinent information came from CDDB.
   Artist/Tittle areas are filled with warning information saying : "Please see https://gnudb.org for more information."
   So, Gnudb.org recommends to downgrade Freac version.
   Unfortunately, I work under CentOs 7 and use snap manager to install Freac and the only version given is 1.1.7.
   Is there a way to reverse back to Freac 1.1.6 using snap installer ? 
   
Thanks for your help,
regards,
Martial

**Comment by CyberBob305** [comment]
Same here... but i did not want to downgrade... is there maybe a temporarily solution to overcome this problem?

**Reply by nmusatti** [reply]
One option is to perform the rip without querying the gnudb database and then use MusicBrainz Picard to retrieve metadata.

**Reply by madum31** [reply]
I abandoned the idea of freac 1.1.6 snap installation.
I downloaded the freac 1.1.6 release, dependencies smooth & boca, and recompiled from sources.
Freac (1.1.6) now works under Centos 7 with correct gnudb exchanges.
I din't have enough knowledge of freac sources to correct the issue.

**Reply by CyberBob305** [reply]
I like my filenames always correspond to the ID3 Tags... and in this case, the filenames were set incorrect.
So i finally downgrade to 1.1.6 (i use the AppImage in Linux Mint)

**Comment by Tyco72** [comment]
Hello. I don't understand what actually happened between freac and Gnudb. It just worked and then no more. Where is the issue? Will it be fixed soon in the next release?

**Comment by 7sundowns** [comment]
Same behavior after ripping 11 CDs. Got also the recommendation to downgrade from 1.17. It was then no longer possible to rip the same CDs again without this error message. Reinstalled fre:ac and restarted the PC what did not help.
Ubuntu 20.04.6 LTS

**Comment by jchv** [comment]
To downgrade the Flatpak version:
```shell
sudo flatpak update --commit=195a236bbd0fe41b9d2533379dcd5d51c488f3466d45cf225cafb4b4565e2e36 org.freac.freac
```
That should give you fre:ac v1.1.6.

**Reply by KolFoxy** [reply]
Amazing, thanks for the help @jchv

**Reply by seanloosier** [reply]
I'm using Debian 11 with snap and was wondering if v1.1.6 could be made available there. Unless there is another way to downgrade which I'm not aware of... I'm kind of new the Linux.

I also see that "snap info freac" shows a development release of 1.1.7 published 10/29 on the edge channel, good to see this! I did try that version, had the same result from gnudb. 

Thanks for all the work!!

**Reply by seanloosier** [reply]
For those having the same issue -

I was able to download the AppImage version and use that. I didn't even have to uninstall the snap version; I'll use this until a new version is available.

Thanks, keep up the good work!

**Comment by Mike-Morrell** [comment]
The explanation on the https://gnudb.org front page says they blocked fre'ac because of bugs causing inaccurate data to be submitted to the database. The fre'ac developer is working on a fix for this.

**Reply by Tyco72** [reply]
Very good. Thank you for the info.

**Comment by cudBwrong** [comment]
To continue using the gnudb feature while the problem is being repaired, downgrade back to freac release 1.1.6.

An easy way to do this is to go to the "releases" page, look for the 1.1.6 section, and click on "assets" which will expand the list of downloads.

https://github.com/enzo1982/freac/releases

You should be able to see a link to the correct download for your system.
